packages/smart-contracts/artifacts/verification/Celo/ERC20FeeProxy_metadata.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
i'm not sure why we need this? (the whole `verification` folder tbh)

Reply:
I have the same feeling about it